app/controllers/api/v1/ping_things_controller.rb

```diff
@@ -18,11 +20,37 @@ def create
         end
       end
 
+      def create_multiple
```

Comment:
```suggestion
      def create_batch
```
to be consistent with the route?